app/styles/Fadein.scss

```diff
@@ -0,0 +1,8 @@
+.fadein-appear {
```

Comment:
nit: since it's not a stylesheet directly related to a react component, the name should be `fadeIn.scss`